src/DataLakeStore/DataLakeStore.Test/ScenarioTests/AdlsTestsBase.cs

```diff
@@ -29,14 +29,15 @@
 using Microsoft.Azure.Management.DataLake.Store;
 using Microsoft.Azure.Commands.DataLakeStore.Models;
 using Microsoft.Azure.ServiceManagement.Common.Models;
+using Microsoft.Azure.Commands.DataLake.Test.ScenarioTests;
 
 namespace Microsoft.Azure.Commands.DataLakeStore.Test.ScenarioTests
 {
     public class AdlsTestsBase : RMTestBase
     {
         private readonly EnvironmentSetupHelper _helper;
 
-        internal const string ResourceGroupLocation = "eastus2";
+        internal const string ResourceGroupLocation = "westus";
```

Comment:
I agree, however this has been hardcoded all along. eastus2/westus are allowed regions. I changed to westus, because, in our dogfood environment, eastus2 ADLS backend was giving issues. That is why changed it. I can try making this change as a part of a separate PR.

Reply:
will create an issue so that I can do this as a part of separate pr.